opensrp-reveal/src/test/java/org/smartregister/reveal/interactor/BaseDrawerInteractorTest.java

```diff
@@ -44,11 +50,21 @@
     @Mock
     private PlanDefinitionSearchRepository planDefinitionSearchRepository;
 
+    @Mock
+    private SQLiteDatabase database;
+
     private BaseDrawerInteractor interactor;
 
     private String planId = UUID.randomUUID().toString();
 
     private String operationalArea = UUID.randomUUID().toString();
+    String syncQuery = "SELECT syncStatus FROM client WHERE syncStatus <> 'Synced'\n" +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5564566)